config/fcos/v1_6_exp/validate_test.go

```diff
@@ -234,3 +234,117 @@ func TestValidateGrubUser(t *testing.T) {
 		})
 	}
 }
+
+func TestValidateMountPoints(t *testing.T) {
+	tests := []struct {
+		in      Config
+		out     error
+		errPath path.ContextPath
+	}{
+		// valid config (has prefix "/etc")
+		{
+			in: Config{
+				Config: base.Config{
+					Storage: base.Storage{
+						Filesystems: []base.Filesystem{
+							{
+								Path: util.StrToPtr("/etc/foo"),
+							},
+						},
+					},
+				},
+			},
+		},
+		// invalid config (path name is '/')
+		{
+			in: Config{
+				Config: base.Config{
+					Storage: base.Storage{
+						Filesystems: []base.Filesystem{
+							{
+								Path: util.StrToPtr("/"),
+							},
+						},
+					},
+				},
+			},
+
+			out:     common.ErrMountPointForbidden,
+			errPath: path.New("yaml", "storage"),
+		},
+		// invalid config (path is /boot)
+		{
+			in: Config{
+				Config: base.Config{
+					Storage: base.Storage{
+						Filesystems: []base.Filesystem{
+							{
+								Path: util.StrToPtr("/boot"),
+							},
+						},
+					},
+				},
+			},
+
+			out:     common.ErrMountPointForbidden,
+			errPath: path.New("yaml", "storage"),
+		},
+		// valid config (has prefix "/var")
+		{
+			in: Config{
+				Config: base.Config{
+					Storage: base.Storage{
+						Filesystems: []base.Filesystem{
+							{
+								Path: util.StrToPtr("/var/foo/bar"),
+							},
+						},
+					},
+				},
+			},
+		},
+		// invalid config (path is nil)
+		{
+			in: Config{
+				Config: base.Config{
+					Storage: base.Storage{
+						Filesystems: []base.Filesystem{
+							{
+								Path: nil,
+							},
+						},
+					},
+				},
+			},
+
+			out:     common.ErrMountPointForbidden,
+			errPath: path.New("yaml", "storage"),
+		},
+		// invalid config (path is )
```

Comment:
missing path

Reply:
my bad ended up just calling it "path is too long" because of the previous code which checked if the length was over 4 in length, regex was a better way to go in the end!